internal/pkg/cli/svc_exec.go

```diff
@@ -286,7 +295,7 @@ func buildSvcExecCmd() *cobra.Command {
 	cmd.Flags().StringVarP(&vars.command, commandFlag, commandFlagShort, defaultCommand, execCommandFlagDescription)
 	cmd.Flags().StringVar(&vars.taskID, taskIDFlag, "", taskIDFlagDescription)
 	cmd.Flags().StringVar(&vars.containerName, containerFlag, "", containerFlagDescription)
-	cmd.Flags().BoolVar(&vars.skipConfirmation, yesFlag, false, yesFlagDescription)
+	cmd.Flags().BoolVar(&vars.skipConfirmation, yesFlag, false, execYesFlagDescription)
```

Comment:
since  `skipConfirmation` is no longer used in any methods can we do this:
```go
func buildSvcExecCmd() *cobra.Command {
   var skipPrompt bool
   ...
   cmd.Flags().BoolVar(&skipConfirmation, yesFlag, false, execYesFlagDescription)
```